containers/ecr-viewer/seed-scripts/create-seed-data.py

```diff
@@ -37,15 +37,10 @@ def _process_files(args):
     requests = []
     folder_paths = []
     configName = "integrated.json"
-    if (
-        os.getenv("CONFIG_NAME") == "AWS_SQLSERVER_NON_INTEGRATED"
-        or os.getenv("CONFIG_NAME") == "AZURE_SQLSERVER_NON_INTEGRATED"
-    ):
+    config = os.getenv("CONFIG_NAME") or ""
+    if config.endswith("NON_INTEGRATED"):
```

Comment:
```suggestion
    if "SQLSERVER" in config:
```
I messed up the logic here 🤦